stockfish/models.py

```diff
@@ -514,7 +517,7 @@ def get_evaluation(self) -> dict:
             elif splitted_text[0] == "bestmove":
                 return evaluation
 
-    def get_top_moves(self, num_top_moves: int = 5) -> List[dict]:
+    def get_top_moves(self, num_top_moves: int = 5, num_nodes: int = 0) -> List[dict]:
```

Comment:
Could you please add the additional argument to the docstring (7 lines below) as well. So something like

> num_nodes: The number of nodes/positions that stockfish searches in the game tree.

and maybe also a short description/example for the `get_top_moves`-function with this parameter to the README.

Reply:
Looks good now :+1: